documentation/developing-your-hyrax-based-app.md

```diff
@@ -269,3 +270,158 @@ proxy_deposit:
 
 If both options exist, whichever option is set from the Administrative Dashboard
 will take precedence.
+
+## Dockerizing a Hyrax Based App for Development
+It is now possible develop a Hyrax based app using a Docker workflow. The process utilizes **docker-compose** to build and run the services necessary for Hyrax based applications.
+
+### Switch to Postgres
+The image that you will build your app's image on uses **Postgres**. If you have just created your Hyrax-based app, for example, you'll need to update your `Gemfile`
+	- `gem "sqlite3", "~> 1.3.0"` ---> `gem "pg"`
+
+### Building the image
+Next you'll want to build the image that your Hyrax container will run from. For that, you will need to create a Dockerfile, build the image from it, and push it to a repository you can fetch from later.
+
+**Build image**
+- Run `cd <hyrax_app_name>`
+- Visit [samveralabs/hyrax](https://hub.docker.com/r/samveralabs/hyrax/tags) and grab the latest image **tag**
+- Run `echo "FROM samveralabs/hyrax:7<tag>" > Dockerfile` to create the Dockerfile
+- Run `docker build .`
+
+**Push image to a repo (assumes dockerhub.io)**
+- On completion, the above command will return an image **sha**, **copy it**
+- Run `docker tag <sha-you-copied> <your-docker-username>/<name-of-your-image>:<tag>`
+- Run `docker push <your-docker-username>/<name-of-your-image>`
+
+### Configure the docker-compose.yml
+Since Hyrax relies on serveral different services to run, you'll need to set up a **docker-compose.yml** to tell Docker how to run all those services together.
+
+- Create a **docker-compose.yml** file your application root
+```
+version: '3.8'
+services:
+  app: &app
+    build:
+      context: .
+      target: hyrax-engine-dev
```

Comment:
does this work? if so, i think maybe it's getting overwritten by `image:` below?

in the hyrax setup, we specify a target because our `Dockerfile` has a multi-stage build, with multiple possible build targets. then we give an `image: ` name to tag the built image for reuse by `db-migrate:` below.

does it work if you try dropping this `target: ` line and replacing each appearance of `image: <your-image-location>` with `image: something-unique-to-this-file`? if so, it might be possible to drop the whole `**Build image**` step for development.